### PR TITLE
feat: support RN 0.63 Pressable in toBeDisabled

### PR DESCRIPTION
**Pull request by mateusbw**
**What**:

Updated jest-native to support RN 0.63 Pressable component in toBeDisabled.

**Why**:

When testing a Pressable component the test always fails. 

**How**:

Updated dependencies to support RN 0.63. Added Pressable to supported Types.

**Checklist**:

- [ ] Documentation added to the
      [docs](https://github.com/testing-library/jest-native/README.md) (N/A)
- [ ] Typescript definitions updated (N/A)
- [X] Tests
- [X] Ready to be merged

Related to issue https://github.com/testing-library/jest-native/issues/23

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "@babel/cli": "7.2.3",
     "@babel/core": "7.4.0",
     "@babel/runtime": "7.4.0",
-    "@testing-library/react-native": "^4.0.1",
+    "@testing-library/react-native": "^6.0.0",
```
Comment (brunohkbx) [inline package.json:42]: Why you had to bump the version?

Comment (mateusbw) [reply package.json:42]: I'm not sure why i had to do it, but before the bump, i was getting this error when running the tests.
> You are trying to `import` a file after the Jest environment has been torn down.

I could only fix it changing the version.

**Comment by brunohkbx** [comment]
For further clarification, updating react-native to `0.63.2` doesn't break tests with `TouchableHighlight` and `TouchableOpacity` as described in #23 because @testing-library/react-native `6.0.0` is mocking them [here](https://github.com/testing-library/native-testing-library/blob/ad41bb61f4b2ef392e15d98a9b49fcaa6af591af/src/preset/mock-modules.js#L69)

**Pull request merged**